docs/topics/DatabaseOperations.adoc

```diff
@@ -300,6 +344,12 @@ The following key derivation functions are supported:
 
  * Argon2 (KDBX 4 - recommended): KDBX 4, the Argon2 key derivation function can be used for transforming the composite master key (as protection against dictionary attacks). The main advantage of Argon2 over AES-KDF is that it provides a better resistance against GPU/ASIC attacks (due to being a memory-hard function). The number of iterations scales linearly with the required time. By increasing the memory parameter, GPU/ASIC attacks become harder (and the required time increases). The parallelism parameter can be used to specify how many threads should be used.

```

Comment:
Need to mention anything about Argon2d and Argon2id? We have those two options in Database Settings but no explanations of them.

Reply:
I thought about it, can easily add a sentence on the difference.